code/game/machinery/camera/camera.dm

```diff
@@ -358,6 +357,16 @@
 	busy = 0
 	return 0
 
+/obj/machinery/camera/proc/Togglelight(on=0)
+	for(var/mob/living/silicon/ai/A in player_list)
```

Comment:
Proc isn't actually called every time the AI eye moves. It only calls it on cameras that it determines need to be turned on if camera lights are on. With one AI it would never call it on a camera that doesn't need to be turned on or off. But yeah, I'll change it to ai_list.